### DISCUSSION TITLE
Self hosted paid version Android mobile unable to login

### DISCUSSION THREAD

**Discussion by PGTBoos** [dani-garcia/vaultwarden#1452] [Q&A]
### Subject of the issue
Unable to login on paid version using android bitwarden app (latest 1/3/2021 google store)
"Java.security.cert.CerthPathValidatorException: Trust anchor for certification path not found.
My credentials are ok, as i can login using chrome and windows, and chrome on android.
Though i cannot make use of the app.

### Deployment environment
Paid hosted internal

* Install method:
by collegue (i think from source) or as a debian package, we're not using docker.

* Clients used: 
LG mobile Thinq G7 with Android 10

* Other relevant details:
Bitwarden latest version was installed using the google store

### Steps to reproduce
Try to login from the app  simply trying to login from the app

### Expected behaviour
Be able to use the app, be able to login

### Actual behaviour
could not logon using the app

### Troubleshooting data
PS i saw an earlier case where this was turned into a discussion.
I intentionally raise it again as a bug, bug should be solved.
Our company is paying for it, so it should be solved.
I fresh new install from the appstore should work.
Going back older versions and reinstall etc.. thats not a solution, fix yo app.

**Comment by BlackDex** [comment]
May i ask who you pay for this? Since this is an OpenSource free to use project. We do however accept donations, but that is not the same.

Also, we have nothing to do with bitwarden.com / 8bit Solutions LLC as is mentioned in our [README.md](https://github.com/dani-garcia/bitwarden_rs/blob/master/README.md)

**Comment by BlackDex** [comment]
Regarding the actual question. It looks like you are using a self-signed certificate, which is something we do not support or encourage to use because of issues like the one you describe.

If you would use a valid certificate which can be provided using acme (LetsEncrypt or ZeroSSL), it would probably solve your problem.
